src/TestFramework/TestFramework/Attributes/TestMethod/TestMethodAttribute.cs

```diff
@@ -9,6 +11,29 @@ namespace Microsoft.VisualStudio.TestTools.UnitTesting;
 [AttributeUsage(AttributeTargets.Method)]
 public class TestMethodAttribute : Attribute
 {
+    private protected readonly struct ScopedCultureDisposable : IDisposable
+    {
+        private readonly CultureInfo _previousCulture;
+        private readonly CultureInfo _previousUICulture;
+
+        public ScopedCultureDisposable(string cultureName)
+        {
+            _previousCulture = CultureInfo.CurrentCulture;
+            _previousUICulture = CultureInfo.CurrentUICulture;
+
+            var newCulture = new CultureInfo(cultureName);
+            // TODO: Should we set both? Should we have different attribute? Same attribute with two arguments?
+            CultureInfo.CurrentCulture = newCulture;
```

Comment:
Important consideration: Validate if that approach is going to work well for tests running in parallel

Reply:
https://github.com/dotnet/runtime/blob/main/src/libraries/System.Private.CoreLib/src/System/Globalization/CultureInfo.cs#L376-L380

Looks to by async level safe, which is ideal for us, because all tests run in their own async context.

This small experiment where I try to "spray" the new culture all over the thread pool seems to confirm that async is properly isolated both on .net and .net framework.

```cs
using System.Collections.Concurrent;
using System.Globalization;


var tsk = Task.Run(async () =>
{
    while (true)
    {
        await Task.Delay(1000);
        Console.WriteLine($"Thread: {Thread.CurrentThread.ManagedThreadId}, Culture: {CultureInfo.CurrentCulture.Name}");
        await Task.Yield();
    }
});

await Task.Delay(1000);

var tasks = new List<Task>();
var threads = new ConcurrentBag<int>();
foreach (var i in Enumerable.Range(0, 1000))
{
    tasks.Add(Task.Run(() =>
    {
        threads.Add(Thread.CurrentThread.ManagedThreadId);
        CultureInfo.CurrentCulture = CultureInfo.GetCultureInfo("cs-CZ");
        Console.WriteLine($"Thread: {Thread.CurrentThread.ManagedThreadId}, Culture: {CultureInfo.CurrentCulture.Name}");
        return Task.CompletedTask;
    }));
}

Task.WaitAll(tasks.ToArray());

Console.WriteLine(string.Join(", ", threads.Distinct()));
Task.WaitAll(tsk);
```